### PR TITLE
Bug fix header

### PR DESCRIPTION
**Pull request by maksim-luzin**

### CODE REVIEW

**Comment by maksim-luzin** [comment]
[https://imgur.com/WkAneXW](url)

**Comment by maksim-luzin** [comment]
https://user-images.githubusercontent.com/64692611/130319217-fb955178-5d48-431b-a439-d84e4ac91f10.mp4

**Comment by maksim-luzin** [comment]
sreencapture-11

File: frontend/src/scenes/Auth/SignupPage/styles.module.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import '../../../styles/mixins', '../../../styles/vars';
+@import "../../../styles/mixins", "../../../styles/vars";
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/Auth/SignupPage/styles.module.scss]: Use single quotes, please

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/CreateProject/components/Layout/styles.module.scss [hunk 2]
```diff
@@ -1,57 +1,56 @@
-@import '../../../../styles/vars';
-@import '../../../../styles/fonts.scss';
+@import "../../../../styles/vars";
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/CreateProject/components/Layout/styles.module.scss]: And here

File: frontend/src/scenes/TrendsPage/styles.module.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
-@import '../../styles/vars';
-@import '../../styles/mixins';
+@import "../../styles/vars";
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/TrendsPage/styles.module.scss]: And here, please

File: frontend/src/scenes/Auth/SignupPage/styles.module.scss [hunk 4]
```diff
@@ -33,8 +33,8 @@
   font-size: $font-size-akcent;
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/Auth/SignupPage/styles.module.scss]: Look like typo (accent)

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/Auth/SignupPage/styles.module.scss (hunk 4 above)
Comment (maksim-luzin) [reply frontend/src/scenes/Auth/SignupPage/styles.module.scss]: This is not my code. But I'll check it out now.

**Pull request merged**